Modules/signalmodule.c

```diff
@@ -267,11 +266,11 @@ report_wakeup_send_error(void* data)
 static void
 trip_signal(int sig_num)
 {
-    _Py_atomic_store_relaxed(&Handlers[sig_num].tripped, 1);
+    _Py_atomic_store_int_relaxed(&Handlers[sig_num].tripped, 1);
```

Comment:
@pitrou: I don't get why Handlers tripped uses "relaxed" order, whereas is_tripped uses "sequentially consistent" order. What's the rationale? Would it be safer to use "sequentially consistent" order  for both?

Reply:
AFAIR, it is because the sequentially consistent `is_tripped` already acts as a barrier and we don't another barrier.

But, yes, using "sequentially consistent" for both would be at least as safe, so feel free to make the change.

cc @gpshead for opinions

Reply:
I would like to suggest to leave it as relaxed as possible since "sequentially consistent" requires more overhead than relaxed.

Reply:
@corona10: If one memory barrier is enough, can you please just add a comment somewhere in the C file to explain the rationale why one variable is accessed with "relaxed" order, and the other "required" "sequentially consistent" order?

Reply:
I changed my mind:
Depending on specific variables, the memory barrier can be dangerous if someone is not familiar with atomic programming or if we lose the history.

To prevent future incidents by human, I changed the implementation to use "sequentially consistent" order for the case.